test/test_charm.py

```diff
@@ -32,47 +33,52 @@
 from .test_helpers import fake_script, fake_script_calls
 
 
-class TestCharm(unittest.TestCase):
+@pytest.fixture
+def tmpdir():
```

Comment:
There's already a built-in `tmpdir` fixture in pytest, right? https://docs.pytest.org/en/6.2.x/tmpdir.html